python/samples/echo/src/api.py

```diff
@@ -9,7 +9,7 @@
 from botbuilder.schema import Activity
```

Comment:
Please follow the samples naming conventions from `js` and `dotnet` (list bot too)